client/src/components/DataFiles/DataFilesSearchbar/DataFilesSearchbar.test.js

```diff
@@ -125,6 +125,43 @@ describe('DataFilesSearchbar', () => {
     expect(getByRole('form')).toBeDefined();
     expect(getByRole('searchbox')).toBeDefined();
     expect(getByTestId('selector')).toBeDefined();
+    expect(getByTestId('summary-of-search-results')).toBeDefined();
+    expect(
+      getByTestId('summary-of-search-results').getAttribute('class')
+    ).not.toMatch(/hidden/);
+    expect(getByPlaceholderText('Search My Data (Frontera)')).toBeDefined();
+  });
+
+  it('has hides results when disabled', () => {
+    const history = createMemoryHistory();
+    history.push(
+      '/workbench/data/api/scheme/system/path?query_string=testquery'
+    );
+    const store = mockStore({
+      systems: systemsFixture,
+      files: {
+        error: {},
+        loading: {},
+      },
+    });
+    const { getByRole, getByTestId, getByPlaceholderText } = renderComponent(
+      <DataFilesSearchbar
+        api="tapis"
+        scheme="private"
+        system="frontera.home.username"
+        resultCount={0}
+        disabled
+      />,
+      store,
+      history
+    );
+
+    expect(getByRole('form')).toBeDefined();
+    expect(getByRole('searchbox')).toBeDefined();
+    expect(getByTestId('selector')).toBeDefined();
+    expect(
+      getByTestId('summary-of-search-results').getAttribute('class')
+    ).toMatch(/hidden/);
```

Comment:
I wanted to use just `.toHaveClass('hidden')` here but it never seemed to work although I use it in the SiteSearchSidebar.test.js.   😕

Reply:
I notice that feature is imported in [`DataFilesToolbar.test.js`](https://github.com/TACC/Core-Portal/blob/272ab4f/client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.test.js#L2-L5). Did you already try that?